### ISSUE TITLE
JSMpeg.Player is much slower than HTML version

### ISSUE THREAD

**Issue by tsunamilx** [phoboslab/jsmpeg#154]
When I use the HTML version:
```html
<div class="jsmpeg" data-url="<url>"></div>
```
I looks very good!

Then I want to control the video by API, so I changed to JS version
```javascript
var player = new JSMpeg.Player(url [, options]);
```

But it is very slow and laggy, my browser is Chrome. Why is that?

BTW, if I use the HTML version, can I get the instance from HTML element, so I can control via the API? Something like:
```javascript
var player = document.querySelector('.jsmpeg');
player.play();
player.stop();
```

**Comment by phoboslab** [comment]
There shouldn't be any performance difference when instantiating JSMpeg directly in JavaScript. After all, the HTML tag uses it internally. Are there any errors in your browser's console?

A way to get the player instance from the HTML tag is a good idea, though. I'll implement this.

**Comment by phoboslab** [comment]
You can now get a reference to the player from the HTML element through `element.playerInstance`. Note that the player's are created in the `DOMContentLoaded` event - they will not be accessible before that.

```
document.addEventListener('DOMContentLoaded', function(){
	var player = document.querySelector('.jsmpeg').playerInstance;
	player.play();
});
```

**Comment by tsunamilx** [comment]
> There shouldn't be any performance difference when instantiating JSMpeg directly in JavaScript. After all, the HTML tag uses it internally.

Yeah, I agree, I also checked the code, there shouldn't be any difference. But it is actually different when I watch the video. I look closely, the visual size of the video of the js version is somehow bigger than the html version (it's like cover vs contain in css background size).
I can upload a screencast here later if you want.

And thanks for the player instance in html version, will be very useful.

**Comment by luizj** [comment]
@phoboslab you have changed min.js version with more than one change, the another code added not work.
